src/main/java/org/broadinstitute/consent/http/models/elastic_search/DatasetTerm.java

```diff
@@ -0,0 +1,170 @@
+package org.broadinstitute.consent.http.models.elastic_search;
+
+import java.util.List;
+import org.broadinstitute.consent.http.models.ontology.DataUseSummary;
+
+public class DatasetTerm {
+
+  private Integer datasetId;
+  private String datasetIdentifier;
+  private String description;
+  private String studyName;
+  private Integer studyId;
+  private Integer participantCount;
```

Comment:
This one is problematic - there could be more than one `FileType` so there isn't just one "participant count". I wasn't sure what to do with it, so I am not filling it out. not sure how to change - could become an array or select the maximum or just get removed.

Reply:
This is a clear mismatch between the new form and the mocks ... leave it as is for now since we'll definitely need revised mocks to show multiple file types per dataset if, indeed, that is what we need.

Reply:
Since there is now an actual participant count, I added code to grab it.